src/context/I18nProvider/I18nProvider.tsx

```diff
@@ -1,4 +1,5 @@
 import get from 'lodash/get'
+import { InterpolationOptions, transformPhrase } from 'node-polyglot'
```

Comment:
Note: We have to import this from `node-polyglot` specifically as `react-polyglot` doesn't reexport the translate function, but rather augments children with `Polyglot.t` from i18n context, see:
https://github.com/nayaabkhan/react-polyglot/blob/master/src/index.d.ts
https://github.com/nayaabkhan/react-polyglot/blob/master/src/translate.js

Given we're not in the context of the augmented children when the `onMissingKey` method is called, there is no way to use `react-polyglot` exports to achieve this. Created https://github.com/nayaabkhan/react-polyglot/issues/67 so that we don't have to switch back and forth between the two packages.